### PR TITLE
1.0.0 - Assimilation to Rally

### PR DESCRIPTION
**Pull request by jeffmay**
@htmldoug @touchdown @patrick-bohan @nicksovich

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 
 name := "scala-enum"
 
-organization := "me.jeffmay"
+organization := "com.rallyhealth"
```
Comment (touchdown) [inline build.sbt]: `com.rallyhealth.core`??

**Comment by touchdown** [comment]
LGTM

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
 
-resolvers += Resolver.url(
-  "bintray-sbt-plugin-releases",
-  url("http://dl.bintray.com/content/sbt/sbt-plugin-releases"))(
-    Resolver.ivyStylePatterns)
-
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
+resolvers += "Rally Plugin Releases" at "https://artifacts.werally.in/artifactory/plugins-release"
 
+addSbtPlugin("com.rallyhealth" %% "rally-versioning" % "0.2.0")
```
Comment (htmldoug) [inline project/plugins.sbt]: HINT 0.2.2 is out.

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,8 +1,6 @@
+// See https://wiki.audaxhealth.com/display/ENG/Build+Structure#BuildStructure-Localconfiguration
+credentials += Credentials(Path.userHome / ".ivy2" / ".credentials")
 
-resolvers += Resolver.url(
-  "bintray-sbt-plugin-releases",
-  url("http://dl.bintray.com/content/sbt/sbt-plugin-releases"))(
-    Resolver.ivyStylePatterns)
-
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.2.1")
+resolvers += "Rally Plugin Releases" at "https://artifacts.werally.in/artifactory/plugins-release"
```
Comment (htmldoug) [inline project/plugins.sbt]: HINT fine for now, but this'll have to change if you open source this.

**Comment by htmldoug** [comment]
LGTM

**Pull request merged**